src/BinSkim.Rules/EnableSpectreMitigations.cs

```diff
@@ -107,8 +104,8 @@ public override void Analyze(BinaryAnalyzerContext context)
                 return;
             }
 
-            TruncatedCompilandRecordList badModuleList = new TruncatedCompilandRecordList();
-            TruncatedCompilandRecordList masmModuleList = new TruncatedCompilandRecordList();
+            TruncatedCompilandRecordList mitigationNotEnabledModules = new TruncatedCompilandRecordList();
```

Comment:
Question--if we're likely to need to expand on the types of states here (e.x. compiled with MASM, etc.), would it make more sense to instead use a structure that maps a state to the appropriate TruncatedCompilandRecordList, rather than continuing to add new lists?

E.x. something like:
Dictionary<State, TruncatedCompilandRecordList>;
Mapping "ExplicitlyDisabled" -> ..., "NotEnabled" -> ..., "MASM" -> ..., etc.